src/js/tabs/pid_tuning.js

```diff
@@ -369,7 +369,7 @@ TABS.pid_tuning.initialize = function (callback) {
             var dValue = parseInt(dElement.val());
             var dMinValue = parseInt(dMinElement.val());
 
-            var dMinLimit = dValue > 0 ? dValue - 1 : 0;
+            var dMinLimit = Math.min(dValue > 0 ? dValue - 1 : 0, 100);
```

Comment:
This looks a bit weird now as we use two different ways to constrain `dvalue`. `Math.min(Math.max(dValue, 0), 100)` would be cleaner.

Reply:
Agreed, just `dValue - 1`